_content/faq.md

```diff
@@ -18,56 +18,56 @@ Here are some Frequently Asked Questions and their answers. This document is sti
-- Client implementation (such as the bare-bones [tus-js-client](https://github.com/tus/tus-js-client#installation) for browsers and Node.js, or if you need a rich UI, [Uppy](https://uppy.io/docs/#getting-started)).
+- server implementation (you can read how to [get started with tusd](https://github.com/tus/tusd/#documentation) which is the most mature option), and a:
+- client implementation (such as the bare-bones [tus-js-client](https://github.com/tus/tus-js-client#installation) for browsers and Node.js, or if you need a rich UI, [Uppy](https://uppy.io/docs/#getting-started).
 
-tus is a low-level building block and there's no one right way to do deploy resumable file uploads with it. Still, to get started quickly, for the majority use case, we'd recommend tusd + Uppy. But the ecosystem is rich and you'll know it if one of our community maintained client/server [implementations](https://tus.io/implementations.html) in Ruby, Python, Elixir, etc, are a better fit to your platform.
+Tus is a low-level building block and there's no one right way to do deploy resumable file uploads with it. Still, to get started quickly for the majority of use cases, we recommend tusd + Uppy. But the ecosystem is rich and you'll know it if one of our community maintained client/server [implementations](https://tus.io/implementations.html) in Ruby, Python, Elixir, etc, are a better fit for your platform.
 
 Finally, if you don't care for self-hosting tus servers, [Transloadit](https://transloadit.com) offers a tus-based upload handling service that can export to the storage of your choice (and can optionally encode or resize files passing through). The setup steps are then to create an account and add an Uppy HTML tag to your website, or check out [non-browser client integrations](https://transloadit.com/docs/#sdks).
 
 ## How does tus work?
 
-This section covers the more low-level details of how tus performs an upload. If you are using one of our open-source [implementations](/implementations.html), you don't have to worry about these details. That being said, they might still come in handy when debugging and troubleshooting.
+This section covers the more low-level details of how tus performs an upload. If you are using one of our open-source [implementations](/implementations.html), you don't have to worry about these details but they may still be helpful to ease debugging and troubleshooting.
 
 A tus upload is broken down into different HTTP requests, where each one has a different purpose:
 
-- At first, the client sends a `POST` request to the server to initiate the upload. This *upload creation request* tells the server basic information about the upload, such as its size or additional metadata. If the server accepts this upload creation request, it will return a successful response with the `Location` header containing the *upload URL*. The upload URL is used to uniquely identify and reference the newly created upload resource.
```

Comment:
In some cases, the backticks (`) were removed. We like to keep them as formatting instruction, so the words in the backticks render as code in the website. Could you undo these deletions? This occurred in multiple files, I think.